test/controllers/income/estimatedPay/update/IncomeUpdateCalculatorControllerSpec.scala

```diff
@@ -305,91 +303,182 @@ class IncomeUpdateCalculatorControllerSpec
     "redirect user for non live employment " when {
       "employment amount is occupation income" in {
         val testController = createTestIncomeUpdateCalculatorController
-        val result: Result = testController.processHowToUpdatePage(
+        val result: Future[Result] = testController.processHowToUpdatePage(
```

Comment:
A lot of the tests in this file share the same setup, would be nice if this could be pulled out in a more generic way. Grab me when you are in and we can discuss some ideas